browser/package.json

```diff
@@ -1,31 +1,31 @@
 {
-  "name": "@0x/mesh-browser",
-  "version": "3.0.1-beta",
-  "description": "TypeScript and JavaScript bindings for running Mesh directly in the browser.",
-  "main": "./lib/index.js",
-  "license": "Apache-2.0",
-  "scripts": {
-    "build": "yarn build:go && yarn build:generate && yarn build:ts",
-    "build:ts": "node --max_old_space_size=4096 ./node_modules/.bin/tsc -b",
-    "build:generate": "go run ./scripts/generate_wasm_buffer.go",
-    "build:go": "GOOS=js GOARCH=wasm go build -o ./wasm/main.wasm ./go/main.go",
-    "docs:md": "ts-doc-gen --sourceDir=./ts --output=${npm_package_config_docsPath}",
-    "lint": "tslint --format stylish --project ."
-  },
-  "config": {
-    "docsPath": "../docs/browser"
-  },
-  "devDependencies": {
-    "@0x/ts-doc-gen": "^0.0.16",
-    "@0x/tslint-config": "^4.0.0",
-    "@types/base64-arraybuffer": "^0.1.0",
-    "tslint": "5.11.0",
-    "typescript": "^3.5.3"
-  },
-  "dependencies": {
-    "@0x/order-utils": "^10.0.1",
-    "@0x/utils": "^5.1.1",
-    "base64-arraybuffer": "^0.2.0",
-    "browserfs": "^1.4.3"
-  }
+    "name": "@0x/mesh-browser",
+    "version": "1.0.0",
```

Comment:
The version here doesn't actually matter since it always get's updated when we run `make cut-release`. I think `1.0.0` makes more sense than `3.0.1-beta`.